### ISSUE TITLE
Add Facet for Work Class Data Type

### ISSUE THREAD

**Issue by roamye** [project-lux/lux-marklogic#140]
**Problem Description:**
A facet to filter for Work class data types such as Visual Work/Textual Work 

**Expected Behavior/Solution:**
Create facet/results for the Works scope and all its values in simple and advanced search. 
These results will pull from the "Type" listed in the Work Record. 

Note: 

Example of record with Textual Work type:
```
{
"id": "https://lux-front-tst.collections.yale.edu/data/text/036a0146-fbd8-4c0b-9c05-6e47d3d3e824",
"type": "LinguisticObject",
"about": [
{
```
Help text will be added for each value. 

**Requirements:**
The following values will be pulled from the Type value in the JSON record. 
- value: Visual Work - 'VisualItem'
- value: Textual Work - 'LinguisticObject'
- The following Help Text will be added for each value selection: 
    - **Work Class:** Search for records categorized as either a "Visual Work" or a "Textual Work.
    - **Visual Work:** Search for Works that are primarily visual, such as the images shown by paintings or photographs, sculptures, or other non language oriented creative expressions.
    - **Textual Work:**  Search for Works that are primarily textual or otherwise convey information via human language. An Object may include both visual and textual works, such as a poster with graphics and text.  In addition, Textual Works encompasses most items from the Library collections, including posters, artworks, musical scores, video and sound recordings.  
    - 
**Needed for promotion:**
If an item on the list is not needed, it should be crossed off but not removed.

- ~[ ] Wireframe/Mockup - Mike~
- [x] Committee discussions - Sarah
- [x] Feasibility/Team discussion - Sarah
- [x] Backend requirements - TBD
- ~[ ] Frontend requirements- TBD~
- [x] Questions
- List of questions for discussions. Answers should be documented within the issue.

**UAT/LUX Examples:**

- Please review UAT ex from FE ticket | https://github.com/project-lux/lux-frontend/issues/177

**Dependencies/Blocks:**
Blocking: https://github.com/project-lux/lux-frontend/issues/177

**Related Github Issues:**

- Issues that contain similar work but are not blocking or being blocked by the current issue.

**Related links:**

- These links can consist of resources, bugherds, etc.

**Wireframe/Mockup:**
Place wireframe/mockup for the proposed solution at end of ticket.

**Comment by clarkepeterf** [comment]
@prowns @roamye @jffcamp 

I have the following for the advanced search label and help text. is this correct?

      label: 'Work Class',
      helpText:
        'Search only records that are either a "Visual Work" or a "Textual Work"',

**Comment by jffcamp** [comment]
label: 'Work Class',
  helpText:
    'Search for records categorized as either a "Visual Work" or a "Textual Work"',

**Comment by roamye** [comment]
In prod, the selection of a work class: visual works or any value does not update for the help text provided above.

after discussion with @clarkepeterf This is because this functionality is not supported at the moment. A new ticket will be created for the new functionality to be supported.

Besides this everything else looks good in PROD. Closing.
